spec/models/media_object_spec.rb

```diff
@@ -772,6 +772,7 @@
       expect(media_object.descMetadata.original_name).to eq 'descMetadata.xml'
     end
```

Comment:
While validating the MODS document, Nokogiri fetches external schema definitions from the LOC website (not so much fun when the website is down).